src/config.ts

```diff
@@ -296,43 +296,44 @@ export class MirrordConfigManager {
     if (this.active) {
       // User has selected a config (via active config button).
       new NotificationBuilder()
-        .withMessage("using active mirrord configuration")
+        .withMessage("Using active mirrord configuration.")
         .withOpenFileAction(this.active)
         .withDisableAction("promptUsingActiveConfig")
         .info();
+
       return this.active;
     } else if (config.env?.["MIRRORD_CONFIG_FILE"]) {
       // Get the config path from the env var.
-      return vscode.Uri.parse(`file://${config.env?.["MIRRORD_CONFIG_FILE"]}`, true);
+      const configFromEnv = vscode.Uri.parse(`file://${config.env?.["MIRRORD_CONFIG_FILE"]}`, true);
+
+      new NotificationBuilder()
+        .withMessage(`Using mirrord configuration from env var "MIRRORD_CONFIG_FILE".`)
+        .withOpenFileAction(configFromEnv)
+        .withDisableAction("promptUsingActiveConfig")
+        .info();
+
+      return configFromEnv;
+
     } else if (folder) {
-      let predefinedConfig = await MirrordConfigManager.getDefaultConfig(folder);
-      if (predefinedConfig) {
+      const configFromMirrordFolder = await MirrordConfigManager.getDefaultConfig(folder);
+
+      if (configFromMirrordFolder) {
         new NotificationBuilder()
-          .withMessage("using a default mirrord config")
-          .withOpenFileAction(predefinedConfig)
+          .withMessage(`Using mirrord configuration from ".mirrord" folder.`)
+          .withOpenFileAction(configFromMirrordFolder)
           .withDisableAction("promptUsingDefaultConfig")
-          .warning();
-        return predefinedConfig;
+          .info();
+
+        return configFromMirrordFolder;
       } else {
         return null;
       }
     } else {
-      folder = vscode.workspace.workspaceFolders?.[0];
-      if (!folder) {
-        throw new Error("mirrord requires an open folder in the workspace");
-      }
+      new NotificationBuilder()
+        .withMessage(`mirrord could not find a configuration file! Try using the "Settings" button or the "Select active config" button to generate/select a mirrord config file.`)
+        .warning();
```

Comment:
Running without a config file should be fine in many cases, so maybe make this `info` instead of `warning` and use less alarming language?

```suggestion
      new NotificationBuilder()
        .withMessage(`mirrord did not find a configuration file. You can use the "Settings" button or the "Select active config" button to generate/select a mirrord config file.`)
        .info();
```

Also, showing a notification when a user runs without creating and selecting a configuration first feels like a significant UI change, so I would like to have @eyalb181's ok on this.

Reply:
No need for a notification here, it's a valid use case to run without a config file

Reply:
Dropped the notification.